### DISCUSSION TITLE
Unable to download video

### DISCUSSION THREAD

**Discussion by lesfans** [aclap-dev/video-downloadhelper#706] [Help and Q&A]
### Your question

Cannot download using the latest beta version, always stuck at 1%，It shows that data is being received, but it seems to always stay above 1%, and the total length of the original video is not displayed. It only displays about 6 seconds.

### Addon & CoApp version and details

version: 9.0.1.22a1
channel: beta
build date: 2024-05-24
build options: linuxlic: false, noyt: false, browser: firefox
lang: zh-CN
license: accepted
key (only 16 first characters): bf1c3d464aae4a59
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "F:\\tools\\tools\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "win7",
    "arch": "x86_64",
    "node": "10"
  },
  "home": "C:\\Users\\ASUS",
  "converterBinary": "F:\\tools\\tools\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
You'll need to supply the URL of a web page hosting a video that you are having a problem with.  Not the name of a web site.  The URL of a specific video.

**Comment by lesfans** [comment]
https://hsex.men/

**Reply by GrampaWildWilly** [reply]
That is exactly what I told you NOT to tell me.  That is the name of a web site.  Give me the URL of a specific video.  That site looks like it's either Chinese or Japanese.  I don't understand either.  Help me a bit here.  Go to a page where there is a video, one video, & post that URL.

**Reply by lesfans** [reply]
OK!
https://hsex.men/video-962726.htm
Thank you!

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I am just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0.1 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

You're running a beta of VDH, same as I am, but you are a step out of date.  You should get the latest version.  You are running the latest CoApp.  You don't appear to be running the latest Firefox.  An update came out earlier today.  You ought to get that.

Oh.  Wait.  It looks like you're still on Windows 7.  OK.  You're running the best Firefox available to you.  I can sympathize.  I only just a few months ago abandoned W7 after more than 10 years using it.  I think some day you will come to realize that you can't stick with the old operating system forever.  For now, I think you can probably get away with using that old Firefox.  Experimentation will prove whether you can.

The URL is, at last, something I can work with.  VDH had no trouble downloading it.  MP4, size 75.9M, resolution 600x336, download time 23 seconds.  It's porn, so I'm not posting my usual array of images to show what happened.  The video played fine in VLC.  I did not sit & watch it, just sampled it at intervals.  It appeared to be there, video & audio, right to the end.

For me, the entry in the VDH menu showed the duration of the video.  It was also labelled as RawHLS.  My inspection of the Network Monitor discovered one HLS manifest but it was not a master manifest, just a stream manifest.  I guess that's why it was RawHLS & not simply HLS.  In any case, VDH downloaded the video without issues.  If you see other entries in the VDH menu, ignore them.  Look for the one with the right duration & type RawHLS.  That's what worked for me.

Once you get the current VDH beta, let's hear how it works for you.

**Reply by lesfans** [reply]
Thank you very much! I'll give you feedback after trying it.
